CONTRIBUTING.md

```diff
@@ -1,10 +1,16 @@
 # Reporting Issues with xDrip
+Before opening an issue, please connect with us by posting in the support channels:  
+https://www.facebook.com/groups/xDripG5  
+https://gitter.im/jamorham/xDrip-plus  
+We will encourage you to open an issue if it is not already known and there is no work-around.
+
 
 ### Please try to provide as much information as possible when raising an issue:
 
 * If it is a bug, please try to explain how to reproduce it, roughly which settings you are using, what you are seeing or not seeing. Screenshots welcomed.
 * If it relates to a new feature, please explain the rationale for the feature, use cases and detail your suggestion.
 
+<br/>
```

Comment:
Please remove all HTML tags unless they are really necessary (i. e. tables).

Reply:
![Annotation 2021-06-14 123545](https://user-images.githubusercontent.com/51497406/121928320-04e0a700-cd0e-11eb-9139-4636d5952790.png)
The image shows what it looks like now.  The red line is the distance between two consecutive paragraphs.  The blue line is the distance between the heading and the content of the same paragraph.    The red line being shorter than the blue line is not very nice.  
I have used the html tag to insert an extra line between paragraphs to make it easier to read and more pleasing to the eye.

Is there another way to add a space?

Reply:
Markdown is a text format an can be read very well as plain text (the main goal). It is not used to style anything. For styling, it is converted to some other formats which supports styles like HTML with CSS. If you look at https://github.com/NightscoutFoundation/xDrip/blob/master/CONTRIBUTING.md it can be read very well. I don't see any reason why the space between the headings and paragraphs should be changed. Nevertheless, it is the GitHub Markdown CSS which is applied to view Markdown files.

Reply:
The reason I added those lines was to make our writings look more professional.  The heading goes with the paragraph it is heading not the previous paragraph.  Look at any article or book.

But, my objective is to close the issue.  So, I have removed the lines as you desire.